src/containers/hoc/with-operations.jsx

```diff
@@ -101,5 +102,10 @@ export const loadData = (options) =>
   compose(
     withOperations(options),
     branch(({ loading }) => loading, renderComponent(LoadingIndicator)),
-    branch(({ errors }) => errors.length > 0, renderComponent(PrettyErrors))
+    branch(
+      ({ errors }) =>
+        errors.length > 0 &&
+        !isOrganizationsPermissionError(errors[0].graphQLErrors[0]),
```

Comment:
question: just for my own clarification, why do we want to branch here if an error is an org permission error?

Reply:
Only superadmins have permissions to query for a list of other organizations, so we expect an error to be thrown for anyone else. The way this `with-operations` component handles that error is with this branch that renders a component just to show the errors when that happens, and not rendering the actual page at all 😞 

So filtering out this specific error is one way we can still load the page as expected for non-superadmins. We haven't formalized it yet (see #1270) but doing it this way is the precedent we set for handling expected permisson errors in #1271